src/libData/AccountData/Account.cpp

```diff
@@ -518,7 +524,8 @@ bool Account::SetInitData(const bytes& initData) {
 
   if (initData.size() == 0) {
     LOG_GENERAL(WARNING, "InitData for this contract is empty");
-    return false;
+    // return false; // change by steve
+    return true;
```

Comment:
How about reverting to returning false here, but only if isScilla is true?

Reply:
Actually this is a bug here, you can't just return true, you need to execute a statement below.   I guess we should just remove the check and assume the empty initData is legit here, and check it when the method is being called in the context of Scilla.